index.bs

```diff
@@ -315,6 +315,30 @@ For example,
 is more readable than
 <code highlight="js">window.scrollBy(50, 0)</code>.
 
+<h3 id="promises">Use Promises</h3>
```

Comment:
Actually, I think what fits best with the rest of the headers is "Design asynchronous APIs using Promises", which appropriately falls between the other two ideas in terms of how much of a command it is.